bin/study.js

```diff
@@ -1,5 +1,25 @@
+'use strict';
+
 // Repeat Ruby Lab for JS
-// let myArray = [1, 2, 3, 4, 5];
+let myArray = [1, 2, 3, 4, 5];
+
+myArray[myArray.length + 1] = 20;
+
+let third = myArray.length - 3;
+myArray[third];
+
+let answerToThree = "this will return nill since there is no ninth element in an array of five elements";
+
+myArray.splice(myArray.length - 5, 3, [-12, -49]); //this doesn't work, but honestly unsure of what is being asked
```

Comment:
actually, you're totally on the right track! I'm not sure how you tested it, but if you array was `[1, 2, 3, 4, 5, , 20]` at this point, this would have produced `[ 1, 2, [ -12, -49 ], , 20 ]`